tests/check-legal-aid/test/ci.json

```diff
@@ -0,0 +1,35 @@
+{
+  "src_folders" : "tests/nightwatch/specs",
+  "output_folder" : "tests/reports/nightwatch",
+  "custom_assertions_path" : "tests/nightwatch/custom_assertions",
+  "custom_commands_path" : "tests/nightwatch/commands",
+
+  "selenium" : {
+    "start_process" : true,
+    "log_path": "tests/reports",
+    "server_path" : "node_modules/selenium-standalone/.selenium/selenium-server/3.7.1-server.jar",
+    "cli_args": {
+      "webdriver.chrome.driver": "node_modules/selenium-standalone/.selenium/chromedriver/2.33-x64-chromedriver"
+    }
+  },
+
+  "test_settings" : {
+    "default" : {
+      "launch_url" : "http://localhost:5000",
```

Comment:
Could we remove this configuration until it's necessary, please?

Reply:
:thumbsup: